### PR TITLE
Add pcbb_xoff parameters for dualtor-aa-56 topo and 50000_40m profile

### PR DESCRIPTION
**Pull request by vivekverma-arista**
### Description of PR

Summary: Add pcbb_xoff parameters for dualtor-aa-56 topo and 50000_40m profile for Arista-7260CX3.
Fixes [#165](https://github.com/aristanetworks/sonic-qual.msft/issues/165)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [x] 202311
- [x] 202405

### CODE REVIEW

**Comment by vivekverma-arista** [comment]
@StormLiangMS and @XuChen-MSFT can you review this?

**Comment by ZhaohuiS** [comment]
@XuChen-MSFT  could you please help review it?

File: tests/qos/files/qos_params.th2.yaml [hunk 1]
```diff
@@ -184,6 +184,34 @@ qos_params:
                 cell_size: 208
             hdrm_pool_wm_multiplier: 4
             cell_size: 208
+        topo-dualtor-aa-56:
+            50000_40m:
+                pcbb_xoff_1:
+                    dscp: 3
+                    ecn: 1
+                    pg: 3
+                    pkts_num_trig_pfc: 19939
```
Comment (XuChen-MSFT) [inline tests/qos/files/qos_params.th2.yaml:193]: Is this value equal with the result of "qos_param_generator.py" for same asic and topology?

Comment (vivekverma-arista) [reply tests/qos/files/qos_params.th2.yaml:193]: Yes.

**Review by XuChen-MSFT — APPROVED** [review]
all good

**Comment by vivekverma-arista** [comment]
@StormLiangMS please help merge

**Review by lolyu — APPROVED** [review]

**Review by zjswhhh — COMMENTED** [review]
Hi @vivekverma-arista - noticed `qos/test_tunnel_qos_remap.py` tests started to fail on arista 7260 acitve-active dualtor after this change.  Can you check? 

```
19:17:03 __init__._fixture_func_decorator         L0073 ERROR  | 
KeyError('100000_300m')
Traceback (most recent call last):
  File "/azp/_work/4/s/tests/common/plugins/log_section_start/__init__.py", line 71, in _fixture_func_decorator
    return fixture_func(*args, **kargs)
  File "/azp/_work/4/s/tests/qos/tunnel_qos_remap_base.py", line 308, in qos_config
    return qos_configs['qos_params'][dut_asic][dut_topo][profile_name]
KeyError: '100000_300m'
ERROR              
```

**Review by zjswhhh — COMMENTED** [review]
Also I didn't see the question `How did you verify/test it?` being answered in the PR description.

**Pull request merged**